MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -316,14 +300,15 @@ store to msg.nmea.lat
 
 
 void read_NMEA(){
+    char string[NMEA_BUF_LENGTH];
     while (Serial.available_S()) {
```

Comment:
As I told before, every time I made change in code, I compile it, flesh it to OSD and test it. How could I know the result in other way? I do not know other way. I do not want to eat out devices memory either, but I do not know even how to do that or avoid that. I am not programmer. I find a way to get some results in code, that's all. You can help and resolve this problem on better way if you wish. Will you?
Regards